### DISCUSSION TITLE
Update field in referenced document

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#945] [Support / Q&A] [answered]
I'm sorry, that should be easy - but I can't "translate" the Firestore-documentation into Rowy...

How do I update a field in a referenced document in an action?
I thought it is easy and should be something like this:

```
    const agentDoc = await row.autoAgentRef.get();
    agentDoc.alertUnseenLinkPotBuyer = 1;

```

or maybe:
`row.autoAgentRef.updateData({"alertUnseenLinkPotBuyer": 1})`

but nothing works :(

Can somebody point me in the right direction please...

Oh, and one more small thing: 
I want to increment the field...the Firebase-documentation says there is "increment(1)" - does that work in rowy too?
Thanks!

**Comment by shamsmosowi** [comment, marked as answer]
https://firebase.google.com/docs/firestore/manage-data/add-data#update-data
`row.autoAgentRef.update({"alertUnseenLinkPotBuyer": 1})` should work for general updates
to increment
```
 const { FieldValue } = require("firebase-admin/firestore");
 const incrementBy1 = FieldValue.increment(1);
 row.autoAgentRef.update({"alertUnseenLinkPotBuyer": incrementBy1})
 ```

**Reply by webwurm** [reply]
Thanks, that works perfectly!
